src/BlockchainReader/Bitcoin.test.ts

```diff
@@ -184,10 +184,17 @@ describe('Bitcoin.dataToPoetAnchor', async assert => {
     })
   }
 
-  assertSample('POET', [2, 3], StorageProtocol.IPFS, 'Jim')
-  assertSample('BARD', [2, 3], StorageProtocol.IPFS, 'Robert')
-  assertSample('POET', [1, 0], StorageProtocol.IPFS, 'Roger')
-  assertSample('BARD', [1, 0], StorageProtocol.IPFS, 'Syd')
+  assert({
+    given: 'A non Poet OP_RETURN Data',
+    should: 'return undefined',
+    actual: dataToPoetAnchor('918cc48177c39a01ab6a20d3e980526775262996de00928fa92d163bd2508812'),
```

Comment:
When reading this test, it's not obvious to me why this data is incorrect.
Can the 'given' be expanded on or the argument made to be easier to see that it's incorrect?